<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="may85" name="Default test" tests="1" failures="1" timestamp="16 Aug 2012 08:56:05 GMT" time="96.997" errors="0">
  <testcase name="testPLT_ACC_010" time="96.945" classname="org.exoplatform.Selenium.platform.eXoGTN.Account.Test_PLT_ACC_010">
    <failure type="java.lang.AssertionError" message="Timeout at waitForElementPresent: //div[text()=&amp;apos;Account Setting&amp;apos; and @class=&amp;apos;MiddleTab&amp;apos;]">
      <![CDATA[java.lang.AssertionError: Timeout at waitForElementPresent: //div[text()='Account Setting' and @class='MiddleTab']
	at org.testng.Assert.fail(Assert.java:89)
	at org.exoplatform.Selenium.Common.CommonHelpers.waitForAndGetElement(CommonHelpers.java:109)
	at org.exoplatform.Selenium.Common.CommonHelpers.click(CommonHelpers.java:226)
	at org.exoplatform.Selenium.UserManagementHelpers.addNewAccountAtNewStaff(UserManagementHelpers.java:31)
	at org.exoplatform.Selenium.platform.eXoGTN.Account.Test_PLT_ACC_010.testPLT_ACC_010(Test_PLT_ACC_010.java:19)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:616)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1197)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1122)
	at org.testng.TestNG.run(TestNG.java:1030)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </failure>
  </testcase> <!-- testPLT_ACC_010 -->
</testsuite> <!-- Default test -->
